### DISCUSSION TITLE
Data Surgeon | Installation Question

### DISCUSSION THREAD

**Discussion by G4k3r** [Drew-Alleman/DataSurgeon#2] [General]
just a quick mention thAT WHEN I put $ ds -h i get no command or like its not even installed

**Comment by Drew-Alleman** [comment]
If you are having an issue with the program you need to open an issue request. Please follow the [bug report template](https://github.com/Drew-Alleman/DataSurgeon/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) if you don't provide enough information the issue _will_ be closed.

**Reply by G4k3r** [reply]
my apologies i am installing rust im sure there is no issue . I request that you make another video walk through from install to first run. Hopefully

**Reply by Drew-Alleman** [reply]
I will work on a video installer. Thanks for the input.

**Reply by Drew-Alleman** [reply]
I added a video if you are still having issues.
